superbench/cli/_help.py

```diff
@@ -32,6 +32,8 @@
     type: command
     short-summary: Deploy the SuperBench environments to all given nodes.
     examples:
+        - name: deploy default image on current GPU node
```

Comment:
same for 'current'.

Reply:
this is a specific example, instead of explaining the usage of argument

Reply:
Agree. If it only mean local node, then we can use local instead of current?

Reply:
updated to "local", thanks